### DISCUSSION TITLE
Help needed with testing on M series Mac

### DISCUSSION THREAD

**Discussion by andrewreeman** [andrewreeman/SpectralSuite#71] [Announcements]
I receive quite a few reports of plugins not being found or crashing on MacOS and all of these seem to come from users with M series Macs.
I've not been able to reproduce these issues on my intel Mac so would appreciate it if anybody (even non-devs) had the time to test these on M series Macs to check for any issues.

If there are then any developers who are able to debug or fix these issues that would also be appreciated. I do suspect that a universal build of the binaries may work but this didn't seem to fix it when previously tried.

**Comment by EskildAnde** [comment]
I've had my daw (fl studio) crash every time i try to open one of the plugins. No freezing or anything like that, it just closes down. Im on a M1 macbook pro. I can send you the crash log if that helps.
